.changelog/1119.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
Curious where `1119` comes from, I see `1116` from your last PR but don't see `1117` or `1118` on main, are they from other PRs?

Reply:
1119 the PR number